### ISSUE TITLE
Windows launcher issue with default Java

### ISSUE THREAD

**Issue by abacefbeca** [HikariKnight/rsu-client#124]
I have to select custom-java and browse to the location of the java.exe in Settings, Script Settings tab, Preferred Java To Use. RuneScape won't start when I click Play Now if default-java is selected.

**Comment by HikariKnight** [comment]
It's a common bug as Oracle do not always update their registry values for Java and sometimes the script can't find the values it needs to guess where Java is (incase a custom path has been chosen for the Java install)

**Comment by abacefbeca** [comment]
I didn't chose a custom path when I installed it and this works
```
C:\Users\User1>java -version
java version "1.8.0_141"
Java(TM) SE Runtime Environment (build 1.8.0_141-b15)
Java HotSpot(TM) Client VM (build 25.141-b15, mixed mode, sharing)
```
so Windows has no trouble finding it. Though, I noticed C:\ProgramData\Oracle\Java\javapath is in the PATH variable instead of C:\Program Files (x86)\Java\jre1.8.0_141\bin and the javapath directory only contains java.exe javaw.exe and javaws.exe.

**Comment by HikariKnight** [comment]
yes but its not for the executable, it is to locate the libraries that the client needs as just launching through the "java" command in general doesnt work because for some reason windows is unable to locate the awt library hence the need to locate where the exe is and grab the dll and force it to load.

**Comment by abacefbeca** [comment]
People will install the launcher in Windows, try starting the game, and think it doesn't work. I suggest making a script that will find the needed Java libraries. Or, at least pop-up a message telling the user they need to find the libraries for the launcher to work. Or, if it does that already then ignore this, I guess. Been a long time since I installed the RSU Launcher for Windows. I thought maybe using JRE_HOME or JAVA_HOME would help. I thought those were set when Java was installed by older installers but AFAICT now you have to set those variables yourself.

**Comment by HikariKnight** [comment]
@abacefbeca will go with the message about asking where java is and will do that as soon as i have some freetime to do so :)
